src/components/Footer/StyledFooter.js

```diff
@@ -0,0 +1,88 @@
+import styled from 'styled-components'
+import { resolveMedia } from '../../utils/helpers'
+import ContainerWrapper from '../Grid/Container/styledContainer'
+import { Row, Col, Container } from '../Grid'
+import RowWrapper from '../Grid/Row/styledRow';
+
+// Main footer container
+const FooterContainer = styled(Container)`
+
+${resolveMedia.md`
+   
+  `}
+
+`;
+
+
+// CONTACT PANEL
+// Social media contacts row
+const ContactPanelRow = styled(Row)`
+    
+
+`;
+
+// Individual social media contact column
+const ContactPanelColumn = styled(Col)`
+  
+ 
+
+`;
+
+
+
+
+
+// GREY PANEL
+// Row for the grey column under the contact panel
+const GreyPanelRow= styled(Row)`
+
+`;
+
+// Social media counters
+const SocialMediaCounter = styled(Col)`
+
+
+`;
+
+// Footer Link
+const FooterLinks = styled(Col)`
+
+
+`;
+// Blog Button and Language Image 
+const Images = styled(Col)`
+
+
+`;
+// Language Text 
+const LanguageText = styled(Col)`
+   
+    ${resolveMedia.md`
+        display: none;
+    `}
+
+`;
+
+
+
+
+// MAS Banner
+// MAS banner row
+const MASBannerRow = styled(Row)`
+ 
+
+`;
+
+
+
+
+export { FooterContainer,
+     GreyPanelRow, 
+     MASBannerRow, 
+     ContactPanelColumn, 
+     ContactPanelRow,
+     FooterLinks,
+     SocialMediaCounter,
+     Images,
+     LanguageText
+    }
```

Comment:
Can we use more descriptive names for all these elements, maybe based on the class names used on RAD Consumer?

Reply:
Done

Reply:
I was thinking more about the names of the containers, though it's not critical. But since this needs to closely follow the current layout of RAD how about using the class names from that? Maybe: 
- FooterContainer
- FooterPrimary (for GreyPanelRow)
- FooterSecondary (for MASBannerRow)
- ContactPanelColumn
- ContactPanelRow
- FooterNav (for FooterLinks)
- SocialMediaLinks (for SocialMediaCounter)
- FooterLinks (for Images)
- Copyright

Reply:
Done Sir